README.md

```diff
@@ -29,8 +29,7 @@ cerberus:
         -    openshift-machine-api
         -    openshift-kube-scheduler
         -    openshift-ingress
-        -    openshift-sdn
-        -    openshift-ovn-kubernetes
+        -    default-sdn
```

Comment:
Thinking if we should add this option outside watch_namespaces as it's not really a namespace? Thoughts?